### ISSUE TITLE
ODF/discreteSample

### ISSUE THREAD

**Issue by kilir** [mtex-toolbox/mtex#41]
Sampling orientations from an odf by

`discreteSample(odf,1000)`

returns

```
Error using  .* 
Matrix dimensions must agree.

Error in quaternion/times (line 21)
     a = a1 .* a2 - b1 .* b2 - c1 .* c2 - d1 .* d2;

Error in ODF/discreteSample (line 24)
ori = orientation(quaternion(S3G_global,r1) .*
quaternion(S3G_local,r2),odf.CS,odf.SS);
```
on mtex-4.0.14.
Changing ODF/discreteSample.m line. 24 to
`ori = orientation(quaternion(S3G_global,r1) .* transpose(quaternion(S3G_local,r2)),odf.CS,odf.SS);`
works for me

**Comment by ralfHielscher** [comment]
Hi kilir,

thank you very much for reporting this. The bug has been fixed in MTEX 4.0.15.

Ralf.

**Comment by kilir** [comment]
Hi Ralf,
it seems to be fixed in ODFComponent/discreteSample.m
 (https://github.com/mtex-toolbox/mtex/commit/8ae8c5486e7ea644b1c5fc5af54e1dc2aa50c6f0)
however I still get the error below which is in ODF/discreteSample.m 
```
startup_mtex

  MTEX 4.0.15 (show documentation)
  Import pole figure data
  Import EBSD data
  Import ODF data

 CS = crystalSymmetry('-3m1')
 odf = unimodalODF(orientation('Euler',[0,5,0].*degree,CS),'halfwidth',5*degree)
oris = discreteSample(odf,500)
Error using  .* 
Matrix dimensions must agree.

Error in quaternion/times (line 21)
     a = a1 .* a2 - b1 .* b2 - c1 .* c2 - d1 .* d2;

Error in ODF/discreteSample (line 24)
ori = orientation(quaternion(S3G_global,r1) .*
quaternion(S3G_local,r2),odf.CS,odf.SS);
```

All the best,
Rüdiger(kilir)
